docs/README.asciidoc

```diff
@@ -40,6 +48,13 @@ Learn how to create your own KeycloakBackup CRD here
 
 === Using external database
 
-The Keycloak Operator allows application developers to utilize an external Postgres database. Learn how to connect an external Postgres database to your KeycloakClient here
+The Keycloak Operator allows application developers to utilize an external Postgres database.
+Learn how to connect an external Postgres database to your KeycloakClient here
 
-* link:external-database.asciidoc[Using external database]
\ No newline at end of file
+* link:external-database.asciidoc[Using external database]
+
+=== Mounting secrets or config maps
```

Comment:
The general idea is to move this documentation to Keycloak Docuemntation repo, see my PR: https://github.com/keycloak/keycloak-documentation/pull/796. Having this in mind, I'm wondering if we need any changes here. I leave it up to you.

Reply:
I can create an additional PR for keycloak-documentation but i think it won't hurt to have some docs in here as well

Reply:
I totally agree!